build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.up.sql

```diff
@@ -137,3 +137,17 @@ ALTER TABLE flows
 ALTER TABLE flows_local
     ADD COLUMN egressName String,
     ADD COLUMN egressIP String;
+ALTER TABLE tadetector
+    ADD COLUMN podNamespace;
```

Comment:
Same above, use comma and add datatype in ADD COLUMNs, please check my previous comment: https://github.com/antrea-io/theia/pull/184#discussion_r1244473267